proxysql-status

```diff
@@ -329,21 +313,35 @@ function parse_args() {
             [[ -n $PROXYSQL_PASSWORD ]] && PASSWORD=$PROXYSQL_PASSWORD;
             [[ -n $PROXYSQL_HOSTNAME ]] && HOST=$PROXYSQL_HOSTNAME;
             [[ -n $PROXYSQL_PORT ]] && PORT=$PROXYSQL_PORT;
-        fi
 
-        # Now override any values with the command-line args
-        if [[ $# -gt 0 && $# -le 4 ]]; then
-            [[ -n ${1+} ]] && USER=$1
-            [[ -n ${2+} ]] && PASSWORD=$2
-            [[ -n ${3+} ]] && HOST=$3
-            [[ -n ${4+} ]] && PORT=$4
-        elif [[ $# -ge 5 ]]; then
-            error "$LINENO" "Incorrect usage"
+        elif [[ $# -eq 0 ]]; then
+            # When no arguments are passed try to read from /etc/proxysql-admin.cnf
+            if [[ ! -r /etc/proxysql-admin.cnf ]]; then
+                error $LINENO "Cannot find /etc/proxysql-admin.cnf to read the credentials." \
+                    "\nYou can either consider creating the cnf file or pass the credentials through command-line in the format <USER> <PASSWORD> <HOST> <PORT>"
+                exit 1
+            else
+                source /etc/proxysql-admin.cnf
+                USER=$PROXYSQL_USERNAME
+                PASSWORD=$PROXYSQL_PASSWORD
+                HOST=$PROXYSQL_HOSTNAME
+                PORT=$PROXYSQL_PORT
+            fi
+            
+        elif [[ $# -gt 4 || $# -lt 4 ]]; then
```

Comment:
-ne 4

Reply:
Thanks for the review. Will update the patch.

Reply:
Fixed